elasticdl/python/tests/pserver_servicer_test.py

```diff
@@ -78,6 +79,8 @@ def create_server_and_stub(
         self._server = pserver.server
         self._stub = elasticdl_pb2_grpc.PserverStub(self._channel)
 
+        self._lr = 0.1
```

Comment:
We set self._lr = 0.1 directly here. Actually, the value is actually set at https://github.com/sql-machine-learning/elasticdl/blob/develop/elasticdl/python/tests/test_module.py#L60

This is a little confusing. We could refine it later.

Reply:
Yes, but we can't get servicer in pserver_servicer_test. Hence, if we want to get *lr*, we need to get model module, and code will be a little tedious. So I set `lr=0.1` directly. 

May set servicer to `parameter_server`?

Reply:
I think it's Okay to add servicer as a member of `ParameterServer`.